src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -69,6 +69,15 @@ def load_arguments(self, _):
         c.argument('features', features_types, options_list=['--features'], help='Space-separated list of features you want to disable.')
         c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
 
+    with self.argument_context('connectedk8s troubleshoot') as c:
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
```

Comment:
Why is 'location' argument reqd? Should not be necessary.

Reply:
It's not required. We can remove, I had earlier added, thinking it might be required. Will remove.